3.10/deployment-kubernetes.md

```diff
@@ -24,4 +24,7 @@ Each of these uses involves a different custom resource.
 
 Continue with [Using the ArangoDB Kubernetes Operator](deployment-kubernetes-usage.html)
 to learn how to install the ArangoDB Kubernetes operator and create
-your first deployment.
\ No newline at end of file
+your first deployment.
+
+For more information about the production readiness state, please refer to the
+[ArangoDB Kubernetes Operator repository](https://github.com/arangodb/kube-arangodb#readme){:target="_blank"}.
```

Comment:
Link directly to the relevant section perhaps?
https://github.com/arangodb/kube-arangodb#production-readiness-state

Reply:
Done.